templates/base.html

```diff
@@ -18,20 +20,24 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-sm-2">
-
       {% block sidebar %}
         {% include "includes/sidebar.html" %}
       {% endblock %}
-
     </div>
-    <div class="col-sm-10 ">
 
-      {% block content %}{% endblock %}
+    <div class="col-sm-10 ">
+      {% block content %}
+        <h1>Car Creation</h1>
+        <form method="POST">
+          {% csrf_token %}
+          {{ form|crispy }}
+          <button type="submit" class="btn btn-primary">Submit</button>
+        </form>
+      {% endblock %}
```

Comment:
The content specific to 'Car Creation' should not be hardcoded in the base template. Move this content to a specific template for car creation to avoid duplication and maintain template reusability.